src/source/JavaGenerator.scala

```diff
@@ -340,6 +340,19 @@ class JavaGenerator(spec: Spec) extends Generator(spec) {
 
         }
 
+        w.wl
+        w.wl("@Override")
+        w.w("public String toString()").braced {
+          w.w(s"return ").nestedN(2) {
+            w.wl(s"\042${self}{\042 +")
+            for (f <- r.fields) {
+              w.wl(s"\042,\042 + \042${idJava.field(f.ident)}=\042 + ${idJava.field(f.ident)} +")
+            }
+          }
+          w.wl(s"\042}\042;")
+        }
```

Comment:
That's not how it works. The triple quotes enclose the entire string and allow single quotes inside. Check the example again.